versioned_docs/version-3.0.0-beta/organization/personal-accounts.md

```diff
@@ -0,0 +1,28 @@
+---
+id: personal accounts
+title: Personal Accounts
+sidebar_label: Personal Accounts
+slug: /organization/personal-accounts
+---
+
+## Personal Accounts
+
+When you create a new account with Spheron, you are creating a Personal Account. A Personal Account is your own, individual account, registered with the Spheron platform and can be upgraded and can act as an admin account for your organization. 
+
+## Understanding the Personal Account and Pro Plan 
+Your Personal Account always exists on the Starter plan and is ideal for personal projects. While you can collaborate with others via pull requests on your repository, you cannot invite others to collaborate on a Team project with a Starter Plan Account. 
+
+<img src={require('@site/static/img/3.0.0/personalaccounts-organization.png').default} />
+
+## Upgrading to a pro plan 
```

Comment:
Remove this part as we don't have option to transfer project from one organisation to other